src/main/resources/application.yml

```diff
@@ -18,7 +18,7 @@ spring:
     max_byte_disk: 10
 
   datasource:
-    url: jdbc:mysql:thin://operations-mysql:3306/IdentityAccountMapper
+    url: jdbc:mysql:thin://operationsmysql:3306/IdentityAccountMapper
```

Comment:
can you make the database name lowercase 'IdentityAccountMapper' as identity_account_mapper